packages/loader/container-loader/src/test/deltaManager.spec.ts

```diff
@@ -510,16 +516,17 @@ describe("Loader", () => {
 						_to: number | undefined,
 						abortSignal?: AbortSignal,
 						_cachedOnly?: boolean,
-					) => {
+					): IStream<ISequencedDocumentMessage[]> => {
 						return {
-							read: async () => {
+							read: async (): Promise<IStreamResult<ISequencedDocumentMessage[]>> => {
 								await new Promise<void>((resolve) => {
 									// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-									abortSignal!.onabort = () => {
+									abortSignal!.addEventListener("abort", () => {
```

Comment:
For reference, I was curious about this one but [it does seem to be equivalent](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal/abort_event#examples).